meetings/2023-01-04.md

```diff
@@ -0,0 +1,46 @@
+# Node.js  Next 10 Years team Meeting 2023-01-04
+
+## Links
+
+* **Recording**:  <http://www.youtube.com/watch?v=j40ZxB-KE4o>
+* **GitHub Issue**: <https://github.com/nodejs/next-10/issues/185>
+
+## Present
+
+* Michael Dawson (@mhdawson)
+* Claudio Wunder (@ovflowd)
+* Jean Burellier (@sheplu)
+* Ruy Adorno (@ruyadorno)
+
+## Agenda
+
+## Announcements
+
+*Extracted from **next10-agenda** labeled issues and pull requests from the **nodejs org** prior to the meeting.
+
+* OpenJS track at Open Source Summit in Vancouver 10-12 in May, planning collaborator summit May 9th, the day before the summit.
+
+### nodejs/next-10
+
+* Metadata Proposal for Docs [#166](https://github.com/nodejs/next-10/issues/166)
+  * Next step was to create demo repository
+  * Claudio walked us through the sample structure, we spent all of the time discussing
+
+We ran out of time and skipped all of the other issues
+* Presentation of the proposal by Claudio [node-doc-proposal](https://github.com/ovflowd/node-doc-proposal)
```

Comment:
I think this part should be up - as we spend a good amount of time talking about that too :)

Reply:
moved